xbmc/guilib/guiinfo/VideoGUIInfo.cpp

```diff
@@ -586,6 +586,9 @@ bool CVideoGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
     case VIDEOPLAYER_AUDIO_LANG:
       value = m_audioInfo.language;
       return true;
+    case VIDEOPLAYER_AUDIOSTREAMCOUNT:
+      value = std::to_string(g_application.GetAppPlayer().GetAudioStreamCount());
+      return true;
```

Comment:
Implementation must return an int, thus belongs into `CVideoGUIInfo::GetInt`. Your Estuary changes are correct. `Integer.IsGreater(...) is what we want.